shared/oae/css/oae.base.css

```diff
@@ -464,14 +464,24 @@ ul.as-selections li.as-original input::-ms-clear {
 }
 
 ul.as-selections li.as-selection-item {
-    border-radius: 3px;
-    box-shadow: none;
+    -webkit-border-radius: 3px;
+       -moz-border-radius: 3px;
+            border-radius: 3px;
+    -webkit-box-shadow: none;
+       -moz-box-shadow: none;
+            box-shadow: none;
     font-family: inherit;
     line-height: 30px;
     margin-bottom: 3px;
     margin-right: 5px;
-    max-width: 400px;
+    max-width: 300px;
```

Comment:
If the span has a `max-width` and a `width: auto`, I don't think the `li` needs them